src/command_modules/azure-cli-network/azure/cli/command_modules/network/commands.py

```diff
@@ -31,7 +34,10 @@
 cli_command(__name__, 'network application-gateway list', custom_path.format('list_application_gateways'))
 cli_command(__name__, 'network application-gateway start', 'azure.mgmt.network.operations.application_gateways_operations#ApplicationGatewaysOperations.start', cf_application_gateways)
 cli_command(__name__, 'network application-gateway stop', 'azure.mgmt.network.operations.application_gateways_operations#ApplicationGatewaysOperations.stop', cf_application_gateways)
-cli_command(__name__, 'network application-gateway show-backend-health', 'azure.mgmt.network.operations.application_gateways_operations#ApplicationGatewaysOperations.backend_health', cf_application_gateways)
+
+if get_api_version(ResourceType.MGMT_NETWORK) in ['2016-09-01']:
```

Comment:
In the case of batch that definitely still looks like something that can be ordered. In the case of preview releases, there is still a heuristic that can be applied to suggest order. 2017-01-01preview would logically come before 2017-01-01 if both existed.

For example, to support network version 2016-12-01 (which is on my plate to do in the near future), I would have to find everywhere these "api_versions in ['2016-09-01']" type statements exist and add '2016-12-01' to them. If at any point I missed one (or it differently slightly so that find wouldn't catch it) now commands will suddenly disappear by virtue of upping the profile. These are exactly the kind of bugs that having "api_greater_than(...)" would prevent from happening.